lib/login-buttons.dart

```diff
@@ -7,6 +8,10 @@ class LoginButton extends StatelessWidget {
   final String text;
   final double padding;
 
+  Future navigateToSignupPage(context) async {
+    Navigator.push(context, MaterialPageRoute(builder: (context) => SignupPage()));
+  }
+
```

Comment:
Any reason this function is defined inside login-register.dart? I feel like this should be defined in main.dart instead, then passed into a LoginButton widget

Reply:
Was just testing out the function. Will move into main.dart!

Reply:
Cool, dont forget to push your changes